gcp/table_gcp_project.go

```diff
@@ -61,6 +62,13 @@ func tableGcpProject(_ context.Context) *plugin.Table {
 				Description: "A list of labels attached to this project.",
 				Type:        proto.ColumnType_JSON,
 			},
+			{
+				Name:        "access_approval",
```

Comment:
As per the API, the column name should be `access_approval_settings`. Also, please update the description to ` The access approval settings associated with a project.